model_compression_toolkit/core/common/mixed_precision/sensitivity_evaluation.py

```diff
@@ -353,6 +353,10 @@ def _get_images_batches(self, num_of_images: int) -> List[Any]:
 
             images_batches.append(inference_batch_input)
             samples_count += batch_size
+        else:
```

Comment:
Seems like the 'else' doesn't have the right indentation, not sure if it's just in the GitHub.. just check that it is aligned with the 'if'